### PR TITLE
document tenant isolation information for message brokers

### PR DESCRIPTION
**Pull request by David-Kunz**

### CODE REVIEW

**Review by sjvans — COMMENTED** [review]
what about message queueing?

**Review by sjvans — APPROVED** [review]

File: node.js/messaging.md [hunk 1]
```diff
@@ -415,6 +415,11 @@ Use [Redis PubSub](https://redis.io/) as a message broker.
 There are no queues:
 - Messages are lost when consumers are not available.
 - All instances receive the messages independently.
+
+::: warning _❗ Warning_
```
Comment (renejeglinsky) [inline node.js/messaging.md:419]:
```suggestion
::: warning No tenant isolation in multitenant scenario
```

File: node.js/messaging.md [hunk 2]
```diff
@@ -444,6 +449,11 @@ Example:
 }
 ```
 
+::: warning _❗ Warning_
```
Comment (renejeglinsky) [inline node.js/messaging.md:452]:
```suggestion
::: warning No tenant isolation in multitenant scenario
```

**Comment by David-Kunz** [comment]
Thanks, @renejeglinsky !

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**